module/x/peggy/keeper/msg_server.go

```diff
@@ -153,6 +153,7 @@ func (k msgServer) RequestBatch(c context.Context, msg *types.MsgRequestBatch) (
 		return nil, err
 	}
 
+	// question: is this right? If i can delegate my voting power to a different key then this would fail each time i call it
```

Comment:
it's being used on the testnet, works with the delegate key, have not tried it with the validator key directly. So I think this is just confusingly written.

Reply:
ah yes, I left some questions in there, I can remove them